python-lib/osisoft_plugin_common.py

```diff
@@ -351,6 +351,16 @@ def get_max_count(config):
     return max_count
 
 
+def reorder_dataframe(unnested_items_rows, first_elements):
+    columns = unnested_items_rows.columns.tolist()
+    for first_element in first_elements:
+        if first_element in columns:
+            columns.remove(first_element)
+            columns.insert(0, first_element)
+    unnested_items_rows = unnested_items_rows[columns]
+    return unnested_items_rows
+
```

Comment:
It's confusing that the first_elements has to be provided in reverse order. Easy to fix but I'd be happy just if a comment mentioned this.

Reply:
Fixed in [c287658](https://github.com/dataiku/dss-plugin-osisoft/pull/35/commits/c287658a84facce059d2e77d62e7be54615d5825)